crates/dojo/world/src/diff/manifest.rs

```diff
@@ -62,6 +63,23 @@ pub struct DojoContract {
     pub systems: Vec<String>,
 }
 
+#[serde_as]
+#[derive(Clone, Default, Debug, Serialize, Deserialize)]
+pub struct DojoLibrary {
+    /// Class hash of the contract.
+    #[serde_as(as = "UfeHex")]
+    pub class_hash: Felt,
+    /// ABI of the contract.
+    pub abi: Vec<AbiEntry>,
```

Comment:
For better readability, I was wondering if it would be better to have the ABIs in a separate object in the manifest. Like so, we could humanly ready the whole manifest ignoring the `abis` key, wdyt? This will be done in other PR, but wondering what's your take on that.